demo/python/cluster-remote-control/client/keypair.pem

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
There are multiple copies of this file in the repo (one for each demo). I think we should move this file to `data` and use the same copy for all demos (because the corresponding public key is hardcoded into the enclave, so accidentally replacing this key with another won't work without also modifying the public key). Maybe this can be fixed in a separate PR.

Reply:
Let's fix this in another PR. I think that'll be cleaner. Too many files being changed in this one.

Reply:
Ok!